examples/plot_bands_functionality.py

```diff
@@ -82,21 +82,6 @@
 ep.plot_bands(array_stack[4], cbar=False)
 plt.show()
 
-##################################################################################
```

Comment:
@nkorinek can you show this with scaling ON in this case rather than removing it?

Reply:
@nkorinek it looks like you missed a comment here. if you can put this text back into the vignette and just show how to turn on scaling that would be great.